### PR TITLE
Use Github actions rather than Buildkite

### PR DESCRIPTION
**Pull request by emosbaugh**

### CODE REVIEW

File: .github/workflows/alpha.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+jobs:
+
+  release:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.17
```
Comment (ricardomaraschini) [inline .github/workflows/alpha.yaml:16]: Any special reason not to go for v1.18 ?

**Comment by emosbaugh** [comment]
[sc-56339]

File: .github/workflows/alpha.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      - uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USER }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+      - run: make build
```
Comment (ricardomaraschini) [inline .github/workflows/alpha.yaml:21]: To build it twice is by design ? I mean, aren't we building it through `deploy/Dockerfile` as well?

File: CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+# Order is important. The last matching pattern has the most precedence.
+# The folders are ordered as follows:
+
+# In each subsection folders are ordered first by depth, then alphabetically.
+# This should make it easy to add new rules without breaking existing ones.
+
+## RULES 
+
+* @replicatedhq/embedded-kubernetes
```
Comment (ricardomaraschini) [inline CODEOWNERS:16]: Seems like this team not yet exist :-(

File: .github/workflows/release.yaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.17
+      - uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USER }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+      - run: echo GIT_TAG=$(echo $GITHUB_REF | awk -F'/' '{ print $NF }') >> $GITHUB_ENV
```
Comment (ricardomaraschini) [inline .github/workflows/release.yaml:21]: I thought we could use `github.ref_name` for that purpose. According to the documentation[1] `ref_name` holds:

```
The branch or tag name that triggered the workflow run.
```

[1] https://docs.github.com/en/actions/learn-github-actions/contexts

File: .github/workflows/alpha.yaml (hunk 1 above)
Comment (laverya) [reply .github/workflows/alpha.yaml:16]: Or 1.19

Comment (emosbaugh) [reply .github/workflows/alpha.yaml:16]: I will bump to 1.19

File: .github/workflows/alpha.yaml (hunk 2 above)
Comment (emosbaugh) [reply .github/workflows/alpha.yaml:21]: nope. it was an oversight

File: CODEOWNERS (hunk 3 above)
Comment (emosbaugh) [reply CODEOWNERS:16]: added

File: .github/workflows/release.yaml (hunk 4 above)
Comment (emosbaugh) [reply .github/workflows/release.yaml]: updated to GITHUB_REF_NAME

File: .github/workflows/alpha.yaml (hunk 2 above)
Comment (emosbaugh) [reply .github/workflows/alpha.yaml]: fixed

File: .github/workflows/alpha.yaml [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+      - uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USER }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+      - run: make docker-image DOCKER_IMAGE=replicated/ekco:alpha
```
Comment (laverya) [inline .github/workflows/alpha.yaml:21]: Should we include the git Sha as version here?

**Review by laverya — APPROVED** [review]

File: .github/workflows/alpha.yaml (hunk 5 above)
Comment (emosbaugh) [reply .github/workflows/alpha.yaml:21]: I dont believe this is necessary https://github.com/replicatedhq/ekco/blob/c1b0d8113a4c13c0b44e981b10ddc35962fda285/Makefile#L6

**Review by ricardomaraschini — APPROVED** [review]

**Pull request merged**